consensus/parlia/parlia.go

```diff
@@ -606,6 +600,9 @@ func (p *Parlia) Prepare(chain consensus.ChainHeaderReader, header *types.Header
 
 	// Set the correct difficulty
 	header.Difficulty = CalcDifficulty(snap, p.val)
+	if header.Difficulty.Cmp(diffInTurn) != 0 && header.Number.Uint64() == 1 {
```

Comment:
Because Parlia doesn't wait for other players for the first block to be produced and it cause the issue when all players are trying to produce the first block at the same time. There are several ways to solve it, one way is to set timer on the validator start to be sure that first block is produced by right validator and other validators have enough time to sync it or just make sure that first block is only produced by only one validator. Otherwise it causes race condition on the first block producing. For other blocks it's not required because there is delays for it.